src/sagemaker/local/local_session.py

```diff
@@ -143,6 +143,10 @@ def create_endpoint(self, EndpointName, EndpointConfigName):
         LocalSagemakerClient._endpoints[EndpointName] = endpoint
         endpoint.serve()
 
+    def update_endpoint(self, EndpointName, EndpointConfigName):
+        del EndpointName, EndpointConfigName
```

Comment:
You don't have to call del here.

Reply:
pylint will complain if EndpointName, EndpointConfigName are not used.

Reply:
Good call!

Let's add this: https://docs.pylint.org/en/latest/faq.html#do-i-have-to-remember-all-these-numbers

You can follow this line: https://github.com/aws/sagemaker-python-sdk/blob/4f08a3d8346b3124d079cb5d84cc9350fc40e400/src/sagemaker/local/local_session.py#L106

Reply:
Please remove the del.